### ISSUE TITLE
Support for a 2 point min max tolerance 

### ISSUE THREAD

**Issue by GitHubDaveSmith** [QualityInformationFramework/qif-community#15]
This would be used to report a diameter characteristic as a 2 point min max evaluation. Currently a custom attribute is used within a DiameterCharacteristicDefinition element.

**Comment by daniel-campbell** [comment]
Is this for an ISO GPS part? I thought that this was possible for ISO GPS -- I don't remember the details, but I thought that the necessary modifiers were added to QIF to handle this. 

But it brings up another point: Y14.5 asks for 2 types of size to be evaluated for a size tolerance - mating envelope and actual local size. It should be possible to specify both results in QIF for a size tolerance, but I'm not sure how this works. This is a topic for investigation.

**Comment by GitHubDaveSmith** [comment]
You are correct, QIF does provide support but it appears to be specifically tailored towards the ISO standard. It would be useful if a solution that is closer harmonised to the ASME standard would be helpful.

**Comment by larry-bergquist** [comment]
Actual local size has been a topic of debate for a long time.  I think there may be clarification in the future.  
For now, I recommend referencing ASME Y14.5.1-2019 (supports Y14.5-2009) para. numbers and titles listed below:
2.3.2 Variation of Size
2.3.2.1 MMC Limit, Where Rule #1 of Y14.5 Applies
2.3.2.2 MMC Limit, Where Rule #1 of Y14.5 Does Not Apply
2.3.3 Actual Local Size Limits
2.3.3.1 Establishing the Local Size Spine
2.3.3.2 Evaluation of Actual Local Size (Opposed Points)
2.3.3.3 Evaluation of Actual Local Size (Circular Elements)
